sources/api/sundog/src/main.rs

```diff
@@ -218,15 +218,10 @@ where
         .context(error::GetPrefixSnafu { prefixes })?;
     debug!("API response model: {}", response.to_string());
 
-    // Build a Settings struct from the response.
-    let settings = serde_json::from_value::<model::Model>(response)
-        .context(error::InterpretModelSnafu)?
-        .settings;
```

Comment:
I am assuming here that because the settings JSON comes from apiclient that it should contain only valid settings, so there's no need to convert to and from Settings just to verify this.